src/pyqasm/visitor.py

```diff
@@ -98,6 +101,7 @@ def __init__(  # pylint: disable=too-many-arguments
         self._clbit_labels: dict[str, int] = {}
         self._alias_qubit_labels: dict[tuple[str, int], tuple[str, int]] = {}
         self._global_qreg_size_map: dict[str, int] = {}
+        self._calibration_qbts_map: dict[str, int] = {}  # Track qubits in defcal and cal blocks
```

Comment:
```suggestion
        self._calibration_qubit_map: dict[str, int] = {}  # Track qubits in defcal and cal blocks
```
Let's keep the full words unless we are going more than 25 chars